### PR TITLE
feat: basic upgrade playbook

### PR DESCRIPTION
**Pull request by aljo242**
closes BLO-1225

### CODE REVIEW

File: upgrades/core/upgrades.go [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+		return fmt.Errorf("invalid params: %w", err)
+	}
+
+	return marketmapKeeper.SetParams(ctx, params)
+}
+
+func setMarketState(ctx sdk.Context, mmKeeper *marketmapkeeper.Keeper, markets marketmaptypes.Markets) error {
+	// markets, err := marketmaptypes.ReadMarketsFromFile("markets.json")
+	//if err != nil {
+	//	return err
```
Comment (Eric-Warehime) [inline upgrades/core/upgrades.go]: Needed?

File: upgrades/core/markets.json [hunk 2]
```diff
@@ -0,0 +1,4056 @@
+[
```
Comment (davidterpay) [inline upgrades/core/markets.json:1]: Is there a reason we need another core market json? We should ideally just have a single central one.

Comment (davidterpay) [reply upgrades/core/markets.json:1]: gonna be a footgun trying to maintain the same file in different places.

File: x/marketmap/types/testdata/marketmap.json [hunk 3]
```diff
@@ -0,0 +1,4058 @@
+{
```
Comment (davidterpay) [inline x/marketmap/types/testdata/marketmap.json:1]: same applies here

File: upgrades/core/markets.json (hunk 2 above)
Comment (aljo242) [reply upgrades/core/markets.json:1]: this is the markets as a list - i can just generate them on the fly

**Pull request closed without merging**